site/data/alerttags.yml

```diff
@@ -1,9 +1,6 @@
 CUSTOM_PAYLOADS:
   link: 
 
-CVE-2012-1823:
-  link: https://nvd.nist.gov/vuln/detail/CVE-2012-1823
```

Comment:
Are these expected?

Reply:
No - will check

Reply:
Is this somehow due to the rule promotions? (Just brainstorming, I have no idea really but that is something that happened recently'ish)

Reply:
The data is regenerated weekly, it would have been caught earlier if it was promotions. Might have been addition of example alerts that miss tags.

Reply:
Does not seem to be that, this CVE is only used in the scan rules 20017 and 20018 which are not yet using example alerts.

Reply:
Seems this needs to be accessed with `getPrivateMethod(…)` https://github.com/zaproxy/zap-admin/blob/e057faa4ae7e24739230831f68c4e9d01144ccae/scripts/generate_alert_pages.js#L230